docs/workforce-management-framework/topics/pro-overview-introduction.adoc

```diff
@@ -0,0 +1,99 @@
+== Introduction
+
+{WFM-RC-NameLong}  consist of set of node.js modules written in TypeScript language that are packaged and distributed as
+{WFM-RC-NameLong} provides a set of reusable components that can simplify mobile, web and cloud application development.
+Solution allows developers to write secure, offline ready websites and mobile applications.
+{WFM-RC-NameLong} provides Workforce Management oriented framework for Workforce Management applications.
+
+Solution consist of two fully independent frameworks:
+
+ * Workflow Management Framework
+ * Rapid Application Development Framework
+
+As extension to frameworks above {ProductName} provides set of example module implementations as part of demo applications.
+
+===  Workflow Management Framework
+
+{WFM-RC-NameLong} Workflow Management Framework build BPMN compliant workflow management solution.
```

Comment:
We aren't `BPMN compliant` etc.  Wort researching how wfm is being described and add it here.